genie-core/src/main/java/com/netflix/genie/core/events/JobFinishedEvent.java

```diff
@@ -45,7 +46,7 @@
     public JobFinishedEvent(
         @NotEmpty final String id,
         @NotNull final JobFinishedReason reason,
-        @NotEmpty final String message,
+        @NotEmpty @Size(max = 255, min = 1) final String message,
```

Comment:
I'm still not sure about the validation magic. I'm sure it's not active in test. In the actual app configuration there is a Validator bean, however I'm not certain in would validate this class.